### ISSUE TITLE
 Maximum call stack size exceeded with "maxNativeZoom" 

### ISSUE THREAD

**Issue by webmatrixxxl** [Mappy/Leaflet-active-area#32]
Hello!

I found out that if "maxNativeZoom" is used and it is exactly 2 less then "maxZoom" when maximum zoom reached - " Maximum call stack size exceeded " is thrown.

```

 L.tileLayer('http://{s}.tile.osm.org/{z}/{x}/{y}.png', {
            attribution: '&copy; <a href="http://osm.org/copyright">OpenStreetMap</a> contributors',
            maxNativeZoom: 14,
            maxZoom: 16,
            minZoom: 11,
        }).addTo(map);


```

It should be exactly "maxZoom" - "maxNativeZoom" === 2 to get the error. I have tested it with your example.

`   "leaflet": "^1.4.0",
    "leaflet-active-area": "^1.1.0"`

**Comment by colbyfayock** [comment]
we're seeing something similar in our project with leaflet `1.5.1`

@paulgreg do you have any advice on this or possible a chance to look at it?

**Comment by colbyfayock** [comment]
hey @webmatrixxxl - @dopeters got a PR merged in that helped fix this issue for us. it's not on NPM yet, however we installed the package via the git repo like so:
```
"leaflet-active-area": "https://github.com/Mappy/Leaflet-active-area.git"
```
hope it helps :)

**Comment by paulgreg** [comment]
Hello,
Yes, sorry for the delay.
Could you confirm that @dopeters’s PR fixes your issue ?
If so, I would generate the npm package.

**Comment by psixdev** [comment]
@colbyfayock @paulgreg
Hello. Any updates? We would not like to use the link to the repository in package.json

**Comment by strimbob** [comment]
"leaflet-active-area": "https://github.com/Mappy/Leaflet-active-area.git"

did not fix it for me

**Comment by colbyfayock** [comment]
@webmatrixxxl does this resolve the issue for you as well?

**Comment by paulgreg** [comment]
Just published release 1.1.1 with the fix.
